### DISCUSSION TITLE
sonoff RM433

### DISCUSSION THREAD

**Discussion by github9876543** [indomus/forum#506] [GENERALE - Discussioni generiche]
Salve,

ho acquistato un telecomando sonoff RM433, ma quando l'ho provato non andava piu' distante di 8 metri
qualcuno ha esperienza di questo oggetto ?
testato con 2 RFR3
testato batteria 12 V ok

grazie

**Comment by KC-inDomus** [comment]
parli del sonoff bridge?

**Reply by github9876543** [reply]
no del telecomando RM433 8 canali
